tools/mo/openvino/tools/mo/back/ChangeOutputTypeAttributes.py

```diff
@@ -85,8 +85,10 @@ def assert_that_is_castable_to_fp16(node: Node):
     original_output = node.out_port(0).data.get_value()
     node.infer(node)
     casted_output = node.out_port(0).data.get_value()
-    original_output_len = len(original_output) if hasattr(original_output, '__len__') else None
-    casted_output_len = len(casted_output) if hasattr(casted_output, '__len__') else None
+    # original_output_len = len(original_output) if hasattr(original_output, '__len__') else None
+    original_output_len = original_output.size
```

Comment:
If `original_output` is None then there is no `size` attribute and there will be an error. 
Please use instead
`original_output.size if hasattr(casted_output, 'size') else None`

Reply:
done